Source/WebCore/Modules/identity/IdentityCredentialsContainer.cpp

```diff
@@ -51,6 +51,11 @@ void IdentityCredentialsContainer::get(CredentialRequestOptions&& options, Crede
         return;
 
     RefPtr document = this->document();
```

Comment:
Do we need a null check?

Reply:
performCommonChecks() checks the document, so probably want an ASSERT here as the document should be guaranteed.